### ISSUE TITLE
[bug]: Cannot overwrite widget vars

### ISSUE THREAD

**Issue by manuxi** [ioBroker/ioBroker.vis-2#373]
### No existing issues.

- [X] There is no existing issue for my problem.

### Describe the bug

If I set a css var, it'll not overwrite the already set var. It will only work if using !important (which should be avoided).

### To Reproduce

Go to your project css, add e.g. `:root {--materialdesign-icon-list-items-text-font-size: 1rem;}`. Use widget "Materialdesign Iconlist". 

### Expected behavior

It's possible to overwrite css vars (which already worked in vis-1).

### Screenshots & Logfiles

---

### Adapter version

2.9.32

### js-controller version

5.0.19

### Node version

v18.17.1

### Operating system

Buanet ioBroker v8

**Comment by foxriver76** [comment]
Can you export the widget? For me it also has no affect with important and as it is an external widget set not really under our control but would like to have a look

**Comment by manuxi** [comment]
Don't know if it would be halpful to export the widget. I'll show you in a different way.
In Vis-1 it was possible to overwrite the widgets var. I've done this so far by use of some css:
![image](https://github.com/ioBroker/ioBroker.vis-2/assets/11303615/05317353-c67f-4480-a61b-ed861ebc3ca2)
This overwrote the widgets value as you can see here:
![image](https://github.com/ioBroker/ioBroker.vis-2/assets/11303615/92016a69-a681-4eca-a533-4503df3a2930)
This is not possible anymote in Vis-2 since the widgets css takes priority. I assume it's because of the order the css is applied.
![image](https://github.com/ioBroker/ioBroker.vis-2/assets/11303615/778f73d9-560b-48c3-8632-d88f6b1a9efc)

**Comment by foxriver76** [comment]
So I am able to let the user css take priority over css however in your specific example I noticed, that `materialdesign-icon-list-items-text-font-size` is set on the element as inline style thus an override will still not work for this specific case without `!important`, has this specific example worked in vis-1? Then the styles of the widget were also applied differently.

![grafik](https://github.com/ioBroker/ioBroker.vis-2/assets/25484431/5c8e45d5-b8a5-47e5-879c-d5ca956ee723)

**Comment by manuxi** [comment]
You're right, this wasn't a good example. I'm used to overwrite vars and saw that I probably did it in Vis-1 (--materialdesign-color-list-item-background).
![image](https://github.com/ioBroker/ioBroker.vis-2/assets/11303615/0cba0ba6-ffef-4b55-b442-e2fce5d33aee)

**Comment by foxriver76** [comment]
Can you try 2.9.35? The user css is now added after the loaded widgets and should have precedence

**Comment by manuxi** [comment]
Seems my project css is gone with v2.9.35.

**Comment by foxriver76** [comment]
what means gone? not applied at all?

**Comment by manuxi** [comment]
Oh, you there ;) Just reverted to v2.9.32.
I also got an js error: moment.duration(...).format is not a function from within a script. It turned out that I had to remove moment-duration-format from the js adapter and added it again. Can't remember having this when switched initially to v2.9.32.

**Comment by manuxi** [comment]
Seemed that my css was completely ignored. 
My Visu went ugly in v2.9.35. I checked with Google dev tools - could not see any project css.
Since I have the Vis-2 in prod, I went back to v.2.9.32.

**Comment by foxriver76** [comment]
Hm https://github.com/ioBroker/ioBroker.vis-2/issues/388 seems like it is broken for "this project"-css

**Comment by manuxi** [comment]
that's what I meant. Sorry, didn't see that issue.

**Comment by foxriver76** [comment]
3.9.36 on the way, will fit it again

**Comment by manuxi** [comment]
The order seems to be correct now (v2.9.36). Thank you very much!
